.rubocop.yml

```diff
@@ -46,3 +46,8 @@ Metrics/ParameterLists:
 
 Style/Lambda:
   Enabled: false
+
+# enable both hash-rocket and ruby19 hashmap syntax
+Style/HashSyntax:
+  Enabled: false
```

Comment:
This is messier than I expected: smart-proxy is on an older version of rubocop which only has 'ruby19_no_mixed_keys". When enabled it insists on changing of hash syntax in most places, a change I'd like to avoid until 1.8.7 support has been dropped completely/unambiguously. Ditto for upgrading of rubocop.

Perhaps I can leave this setting disabled for now? Alternatively you could switch to 1.8 syntax in bundler file.